### PR TITLE
chore: update to indy-sdk-react-native

### PR DESCRIPTION
**Pull request by TimoGlastra**
Signed-off-by: Timo Glastra <timo@animo.id>

Use indy-sdk-react-native instead of rn-indy-sdk. Best to wait until https://github.com/hyperledger/indy-sdk-react-native/pull/14 is merged

### CODE REVIEW

File: packages/react-native/package.json [hunk 1]
```diff
@@ -29,17 +29,17 @@
...
   "devDependencies": {
     "@types/react-native": "^0.64.10",
-    "@types/rn-indy-sdk": "npm:@types/indy-sdk@^1.16.5",
+    "@types/indy-sdk-react-native": "npm:@types/indy-sdk@^1.16.6",
     "react": "17.0.1",
     "react-native": "0.64.2",
     "rimraf": "~3.0.2",
     "typescript": "~4.3.0",
-    "rn-indy-sdk": "^0.1.11",
+    "indy-sdk-react-native": ">= 0.1.13",
```
Comment (blu3beri) [inline packages/react-native/package.json:37]: Would it not be better if we used `^0.1.13`? This way we don't update to a new major release accidentally.

File: packages/react-native/package.json [hunk 2]
```diff
@@ -29,17 +29,17 @@
...
     "react-native-get-random-values": "^1.7.0",
     "react-native-fs": "^2.18.0"
   },
   "peerDependencies": {
-    "rn-indy-sdk": ">= 0.1.11",
+    "indy-sdk-react-native": ">= 0.1.13",
```
Comment (blu3beri) [inline packages/react-native/package.json:42]: Same.

File: packages/react-native/package.json (hunk 1 above)
Comment (TimoGlastra) [reply packages/react-native/package.json:37]: Agree.

**Comment by TimoGlastra** [comment]
Ready for review. Before merging i'd like to test it out in one of our mobile apps to see if everything works as expected

File: yarn.lock [hunk 3]
```diff
@@ -2154,13 +2154,6 @@
   dependencies:
     "@types/node" "*"
```
Comment (blu3beri) [inline yarn.lock:2155]: Should that not be a dev dependency?

Comment (TimoGlastra) [reply yarn.lock:2155]: yarn.lock doesn't discriminate between dev dependencies and non dev dependencies. Package.json is correct i think

**Review by blu3beri — APPROVED** [review]

**Review by jakubkoci — APPROVED** [review]

**Review by JamesKEbert — APPROVED** [review]
All looks good to me! I haven't had the opportunity to test installation via the package as of yet though. Should be fairly straightforward though, so I can do so and report back here if desired.

**Comment by TimoGlastra** [comment]
I've tested the package directly (without AFJ) and everything seemed to work fine. Could create wallets, open pools
Etc...

**Pull request merged**